data/src/test/java/org/apache/iceberg/data/DeleteReadTests.java

```diff
@@ -51,18 +54,32 @@
       Types.NestedField.required(2, "data", Types.StringType.get())
   );
 
+  public static final Schema DATE_SCHEMA = new Schema(
+          Types.NestedField.required(1, "dt", Types.DateType.get()),
+          Types.NestedField.required(2, "data", Types.StringType.get()),
+          Types.NestedField.required(3, "id", Types.IntegerType.get())
```

Comment:
Indentation is off in this file. Please match the existing indentation, where continuation indents are 4 spaces.